jest.config.js

```diff
@@ -1,26 +1,19 @@
-/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+const esModules = [
+  '@juggle/resize-observer',
+  '@react-hook',
+  'lodash-es',
+  'nanoid',
+].join('|')
+
+/** @type {import('jest').Config} */
 module.exports = {
-  preset: 'ts-jest',
```

Comment:
Probably didn't need to remove ts-jest but I think it will speed things up, and simplify the configuration here